src/type/directives.js

```diff
@@ -84,10 +86,13 @@ export class GraphQLDirective {
           name: argName,
           description: arg.description === undefined ? null : arg.description,
           type: arg.type,
-          defaultValue: arg.defaultValue
+          defaultValue: arg.defaultValue,
+          astNode: arg.astNode || null,
         };
       });
     }
+
+    this.astNode = config.astNode || null;
```

Comment:
Could you move this to align with the other `this.xxx =` statements above? Also this should not use `|| null`